tests/end_to_end/test_emotet/data/test_cases/cases_emotet.yaml

```diff
@@ -0,0 +1,16 @@
+- name: emotet_attack
+  description: Execute emotet attack and check generated alerts
+  configuration_parameters: null
+  metadata:
+    regsvr32:
+      rule.id: 255561
+      rule.level: 5
+      rule.description: 'Regsvr32: C:\\\\\\\\Windows\\\\\\\\System32\\\\\\\\wscript.exe'
```

Comment:
Is it necessary to have so many `\`?

Reply:
Yes, so many are needed because of Windows, YAML, and regex.